<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active/>
    <inactive/>
    <disabled/>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Narozeniny</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icon/mainIcon</normaloff>:/icon/mainIcon</iconset>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOsoby">
    <property name="title">
     <string>Osoby</string>
    </property>
    <addaction name="actionAddPerson"/>
    <addaction name="actionEditPerson"/>
    <addaction name="separator"/>
    <addaction name="actionExitApp"/>
   </widget>
   <widget class="QMenu" name="menuNapoveda">
    <property name="title">
     <string>Napoveda</string>
    </property>
    <addaction name="actionAboutQtHelp"/>
    <addaction name="actionAboutHelp"/>
   </widget>
   <widget class="QMenu" name="menuZobrazit">
    <property name="title">
     <string>Zobrazit</string>
    </property>
    <addaction name="actionViewToolbar"/>
    <addaction name="actionViewStatusBar"/>
   </widget>
   <addaction name="menuOsoby"/>
   <addaction name="menuZobrazit"/>
   <addaction name="menuNapoveda"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAddPerson"/>
   <addaction name="actionEditPerson"/>
   <addaction name="separator"/>
   <addaction name="actionAboutQtHelp"/>
   <addaction name="actionAboutHelp"/>
   <addaction name="separator"/>
   <addaction name="actionExitApp"/>
  </widget>
  <action name="actionAddPerson">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/resources/icon/addPerson.png</normaloff>:/resources/icon/addPerson.png</iconset>
   </property>
   <property name="text">
    <string>Pridat novou</string>
   </property>
   <property name="statusTip">
    <string>Vytvori novou osovu nebo udalost</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionEditPerson">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/resources/icon/editPerson.png</normaloff>:/resources/icon/editPerson.png</iconset>
   </property>
   <property name="text">
    <string>Upravit</string>
   </property>
   <property name="statusTip">
    <string>Upravi existujici osovu nebo udalost</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionExitApp">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/resources/icon/exit.png</normaloff>:/resources/icon/exit.png</iconset>
   </property>
   <property name="text">
    <string>Ukoncit</string>
   </property>
   <property name="statusTip">
    <string>Ulozi a ukonci program</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionViewToolbar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Panel nastroju</string>
   </property>
   <property name="statusTip">
    <string>Prepina skryvani nastrojoveho radku</string>
   </property>
  </action>
  <action name="actionViewStatusBar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Stavovy radek</string>
   </property>
   <property name="statusTip">
    <string>Prepina zobrazovani statusoveho radku</string>
   </property>
  </action>
  <action name="actionAboutQtHelp">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/resources/icon/qt.png</normaloff>:/resources/icon/qt.png</iconset>
   </property>
   <property name="text">
    <string>O Qt</string>
   </property>
   <property name="statusTip">
    <string>Zobrazi napovedu o frameworku Qt.</string>
   </property>
  </action>
  <action name="actionAboutHelp">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/resources/icon/about.png</normaloff>:/resources/icon/about.png</iconset>
   </property>
   <property name="text">
    <string>O programu</string>
   </property>
   <property name="toolTip">
    <string>O programu</string>
   </property>
   <property name="statusTip">
    <string>Zobrazit napovedu o programu</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
